app/backend/approaches/chatreadretrieveread.py

```diff
@@ -368,15 +366,14 @@ def run(self, history: Sequence[dict[str, str]], overrides: dict[str, Any]) -> A
             #print("Few Shot Tokens: ", self.num_tokens_from_string(self.response_prompt_few_shots[0]['content'], "cl100k_base"))
             #print("Message Tokens: ", self.num_tokens_from_string(message_string, "cl100k_base"))
 
-            chat_completion = openai.ChatCompletion.create(
+            chat_completion = await openai.ChatCompletion.create(
```

Comment:
use await with acreate variant. "await openai.ChatCompletion.acreate" 

same on line 170 when we make first call to open AI just to be consistent.

Reply:
Updated!